starknet_py/net/client_models.py

```diff
@@ -711,7 +724,28 @@ class SierraContractClass:
     contract_class_version: str
     sierra_program: List[str]
     entry_points_by_type: SierraEntryPointsByType
-    abi: Optional[str] = None
+    raw_abi: Optional[str] = None
```

Comment:
Maybe let's leave the name `abi` here to align with [the RPC returned type](https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L3291)? Then we can rename the method to something like `parse_abi`/`parsed_abi`.

Reply:
I added it from perspective `get_class_by_hash`, the goal was to make api `ContractClass` and `SierraContractClass` as similar as possible. TBH I haven't got a strong opinion which approach I prefer 
```
    async def get_class_by_hash(
        self, class_hash: Hash
    ) -> Union[ContractClass, SierraContractClass]:
```

Reply:
It also makes sense. @THenry14 @DelevoXDG what are your thoughts on this?